src/processTargets/targetUtil/insertionRemovalBehaviors/DelimitedSequenceInsertionRemovalBehavior.ts

```diff
@@ -5,15 +5,10 @@ import { Target } from "../../../typings/target.types";
  * Constructs a removal range for the given target that includes either the
  * trailing or leading delimiter
  * @param target The target to get the removal range for
- * @param contentRange Can be used to override the content range instead of
- * using the one on the target
  * @returns The removal range for the given target
  */
-export function getDelimitedSequenceRemovalRange(
-  target: Target,
-  contentRange?: Range
```

Comment:
This parameter was confusing, and only used in one place, and removing it didn't break any tests, so I removed it.  Fwiw the place where we were using it was actually broken anyway 😅

Reply:
This is a regression. Granted in some weird edge cases. Unfortunately we were lacking tests.
in markdown for example
`"# Name "`
`chuck name` will on main remove that trailing white space. Your changes will leave the white space.